### PR TITLE
fix: Wrap up Mana Calculator

### PR DESCRIPTION
**Pull request by marc2332**
Closes https://github.com/iotaledger/iota-wiki/issues/1340

### CODE REVIEW

File: src/components/ManaCalculator/components/OutputForm.tsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 import humanizeDuration from 'humanize-duration';
 
 export function OutputForm() {
   const { state } = useManaState();
   const results = useResults(state);
   const passiveRewards = roundMax(fromMicro(results.passiveRewards), 6);
   const manaGenerated = roundMax(fromMicro(results.generatedRewards), 6);
-  const totalBPS = roundMax(results.totalBPS, 2);
+  const blockAllowance = roundMax(results.blockAllowance, 2);
+  const totalBPS = roundMax(results.totalBPS, 20);
```
Comment (marc2332) [inline src/components/ManaCalculator/components/OutputForm.tsx:13]: Do we really want to have up to 20 decimal numbers? Perhaps you wanted to use `2` ? @oliviasaa

**Review by begonaalvarezd — DISMISSED** [review]
🚀

File: docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md [hunk 2]
```diff
@@ -6,35 +6,42 @@ Mana is a reward resource generated by holding IOTA tokens. It is utilized for b
...
 <ManaCalculator/>
 
 
 ## Advanced settings
 
-The steps outlined above can give you a rough estimate of your Mana generation and granted BPS. However, playing with the advanced settings will give you more precise results. 
+The steps outlined above can give you a rough estimate of your Mana generation and granted blocks allowance. However, playing with the advanced settings will give you more precise results. 
 
-- **Change the state of the system:** The calculator's default settings assume a certain level of participation in the consensus (i.e., locked stake, delegated stake, and performance factor of validators). Those settings can be changed under the “Advanced Settings - State of the System” tab. You can also add or delete validators and change their fixed costs. 
+- **Change the state of the system:** The calculator's default settings assume a certain level of participation in the consensus (i.e., locked stake, delegated stake, and performance factor of validators). Those settings can be changed under the “Advanced Settings - Validators” tab. You can also add or delete validators and change their fixed costs.
```
Comment (Dr-Electron) [inline docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md:33]:
```suggestion
- **Change the system’s state:** The calculator’s default settings assume a certain level of participation in the consensus (i.e., locked stake, delegated stake, and the performance factor of validators). Those settings can be changed under the “Advanced Settings - Validators” tab. You can also add or delete validators and change their fixed costs. 
```

File: docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md [hunk 3]
```diff
@@ -6,35 +6,42 @@ Mana is a reward resource generated by holding IOTA tokens. It is utilized for b
...
+- If you choose to delegate, you can change two default values under the "Delegator Settings" tab in "Role Configuration":
+   - **Change the validator you delegate to:** Under the default settings, you'll delegate to Validator 1 (which is  pool that, technically speaking, is under the "equilibrium state", see the Whitepaper for more details). However, you can change this setting and know your rewards if you participate in other pools, with different share of delegated to locked stake, and different performance factors.
+  - **Change the number of delegated tokens:** By default, you will delegate all your tokens to the pool selected above.
+   However, you can also simulate a scenario where you only delegate part of your tokens, with the rest remaining undelegated.
+   As usual, pay attention to the units! The input should be done in IOTA or Shimmer (not microIota or Glow).
```
Comment (Dr-Electron) [inline docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md:39]:
```suggestion
- If you choose to delegate, you can change two default values under the "Delegator Settings" tab in "Role Configuration":
   - **Change the validator you delegate to:** Under the default settings, you'll delegate to Validator 1 (which is a pool that, technically speaking, is under the "equilibrium state"; see the Whitepaper for more details). However, you can change this setting and know your rewards if you participate in other pools, with different shares of delegated to locked stakes and different performance factors.
  - **Change the number of delegated tokens:** By default, you will delegate all your tokens to the pool selected above.
   However, you can also simulate a scenario where you only delegate part of your tokens, with the rest remaining undelegated.
   As usual, pay attention to the units! The input should be done in IOTA or Shimmer (not microIota or Glow).
```

File: docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md [hunk 4]
```diff
@@ -6,35 +6,42 @@ Mana is a reward resource generated by holding IOTA tokens. It is utilized for b
...
 
-- If you choose to delegate:
-   - **Change the validator you delegate to:** Under the default settings, you'll delegate to Validator 1 (which is  pool that, technically speaking, is under the "equilibrium state", see the WP for more details). However, you can change this setting and know your rewards if you participate in other pools, with different share of delegated to locked stake, and different performance factors.
+- If you choose to delegate, you can change two default values under the "Delegator Settings" tab in "Role Configuration":
+   - **Change the validator you delegate to:** Under the default settings, you'll delegate to Validator 1 (which is  pool that, technically speaking, is under the "equilibrium state", see the Whitepaper for more details). However, you can change this setting and know your rewards if you participate in other pools, with different share of delegated to locked stake, and different performance factors.
```
Comment (Dr-Electron) [inline docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md:36]: Should we link the whitepaper?

File: docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md [hunk 5]
```diff
@@ -6,35 +6,42 @@ Mana is a reward resource generated by holding IOTA tokens. It is utilized for b
...
+   As usual, pay attention to the units! The input should be done in IOTA or Shimmer (not microIota or Glow).
+   - **Change your performance factor:** Your performance factor under the calculator's default settings is 1.0. However, you can change this to simulate smaller shares of online time. 
+   - **Change your fixed costs:** By the default settings of the calculator, your fixed costs are zero. This setting can be changed so you spare part of the pool rewards just for yourself. However, notice that the fixed cost is a public parameter, that you define during registration. This means that the delegator knows how you set this value and might consider it when choosing a validator to delegate. Furthermore, if you set this value too high, you'll be punished and won't get any rewards.
+   - **Change the amount of stake delegated to you:** The calculator's default settings automatically assign you a certain share of the delegated stake when you start validating, based on a certain "equilibrium state" (see the Whitepaper for more details). However, you can change this setting to know your rewards should you manage to attract more (or less) delegated stake than in the default setting.
```
Comment (Dr-Electron) [inline docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md:47]:
```suggestion
    - **Change the number of staked tokens:** As in the case of delegators, by default, you will stake all your tokens.
   However, you can also simulate a scenario where you only stake part of your tokens, with the rest remaining unstaked.
   As usual, pay attention to the units! The input should be done in IOTA or Shimmer (not microIota or Glow).
   - **Change your performance factor:** Under the calculator's default settings, your performance factor is 1.0. However, you can change this to simulate smaller shares of online time. 
   - **Change your fixed costs:** By the calculator's default settings, your fixed costs are zero. This setting can be changed so you can spare part of the pool rewards for yourself. However, notice that the fixed cost is a public parameter you define during registration. This means that the delegator knows how you set this value and might consider it when choosing a validator to delegate. Furthermore, if you select this value too high, you'll be punished and won't get any rewards.
   - **Change the amount of stake delegated to you:** The calculator's default settings automatically assign you a particular share of the delegated stake when you start validating, based on a specific "equilibrium state" (see the Whitepaper for more details). However, you can change this setting to know your rewards should you attract more (or less) delegated stakes than in the default setting.
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]
LGTM just some fixes and suggestions in the documentation

File: docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md [hunk 6]
```diff
@@ -6,35 +6,42 @@ Mana is a reward resource generated by holding IOTA tokens. It is utilized for b
...
+   As usual, pay attention to the units! The input should be done in IOTA or Shimmer (not microIota or Glow).
+   - **Change your performance factor:** Under the calculator's default settings, your performance factor is 1.0. However, you can change this to simulate smaller shares of online time. 
+   - **Change your fixed costs:** By the calculator's default settings, your fixed costs are zero. This setting can be changed so you can spare part of the pool rewards for yourself. However, notice that the fixed cost is a public parameter you define during registration. This means that the delegator knows how you set this value and might consider it when choosing a validator to delegate. Furthermore, if you select this value too high, you'll be punished and won't get any rewards.
+   - **Change the amount of stake delegated to you:** The calculator's default settings automatically assign you a particular share of the delegated stake when you start validating, based on a specific "equilibrium state" (see the Whitepaper for more details). However, you can change this setting to know your rewards should you attract more (or less) delegated stakes than in the default setting.
```
Comment (Dr-Electron) [inline docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md:47]:
```suggestion
   - **Change the amount of stake delegated to you:** The calculator's default settings automatically assign you a particular share of the delegated stake when you start validating, based on a specific "equilibrium state" (see the [Whitepaper](https://files.iota.org/papers/IOTA_2.0_Incentives_And_Tokenomics_Whitepaper.pdf) for more details). However, you can change this setting to know your rewards should you attract more (or less) delegated stakes than in the default setting.
```

File: docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md (hunk 4 above)
Comment (Dr-Electron) [reply docs/learn/protocols/iota2.0/core-concepts/mana-calculator.md]: Done

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**